spec/models/bucket_list_spec.rb

```diff
@@ -8,6 +8,9 @@
   it { should_not have_valid(:title).when(nil, '') }
   it { should validate_uniqueness_of(:title) }
 
+  it { should have_valid(:description).when('Our fist trip', 'South East Asia with Kam') }
```

Comment:
Line is too long. [90/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.